include/exadg/postprocessor/error_calculation.cpp

```diff
@@ -119,57 +114,44 @@ ErrorCalculator<dim, Number>::setup(dealii::DoFHandler<dim> const &   dof_handle
   mapping     = &mapping_in;
   error_data  = error_data_in;
 
+  time_control.reinit(
+    error_data_in.analytical_solution_available, // TODO: do we realy always activate error calc if
```

Comment:
Yes, I think this is the meaning of this variable. Maybe it is better if you rename this variable into `calculate` as for the other postprocessing modules.

Reply:
In the application files, it might still be useful to call this function `analytical_solution_available`, and doing the assignment `error_data.calculate = analytical_solution_available;`